core/src/main/java/jenkins/util/JSONSignatureValidator.java

```diff
@@ -73,7 +75,17 @@ public FormValidation verifySignature(JSONObject o) throws IOException {
                     certs.add(c);
                 }
 
-                CertificateUtil.validatePath(certs, loadTrustAnchors(cf));
+                try {
+                    CertificateUtil.validatePath(certs, loadTrustAnchors(cf));
+                } catch (GeneralSecurityException | IOException e) {
```

Comment:
You are right, I don't know why I overwritted the java.level value to 1.7 (https://github.com/jenkinsci/jenkins/blob/master/pom.xml#L616 )